src/patterns/components/message/overview.hbs

```diff
@@ -0,0 +1,18 @@
+---
+order: 1
+notes: |
+  Message pattern
+---
+
+<div class="Message {{messageClass}}">
+  {{#block "content"}}
+    <p>Hi, I'm a message!</p>
+  {{/block}}
+  {{#if includeClose}}
+    <button class="Message-close">
+      {{#svg "icons/close" aria-labelledby=(defaultTo closeIconTitle "Message-close-title")}}
+        <title id="Message-close-title">Close</title>
```

Comment:
Gah, sorry, my mistake: the `id` value should be `{{defaultTo closeIconTitle "Message-close-title"}}`. No use supporting a custom title if I don't actually use it. (facepalm)